src/main/utils/pomTree.ts

```diff
@@ -0,0 +1,88 @@
+import { MavenUtils } from './mavenUtils';
+import { TreesManager } from '../treeDataProviders/treesManager';
+import { ContextUtils } from './contextUtils';
+import * as path from 'path';
+
+export class PomTree {
+    constructor(
+        private _pomGav: string = '',
+        private _pomPath: string = '',
+        private _children: PomTree[] = [],
+        private _parent?: PomTree,
+        private _parentGav: string = ''
+    ) {}
+
+    public get pomGav(): string {
+        return this._pomGav;
+    }
+
+    public get pomPath(): string {
+        return this._pomPath;
+    }
+
+    public get children(): PomTree[] {
+        return this._children;
+    }
+
+    public get parent(): PomTree | undefined {
+        return this._parent;
+    }
+
+    public get parentGav(): string {
+        return this._parentGav;
+    }
+
+    public set pomGav(v: string) {
+        this._pomGav = v;
+    }
+
+    public set pomPath(v: string) {
+        this._pomPath = v;
+    }
+
+    public set children(v: PomTree[]) {
+        this._children = v;
+    }
+
+    public set parent(v: PomTree | undefined) {
+        this._parent = v;
+    }
+
+    public set parentGav(v: string) {
+        this._parentGav = v;
+    }
+
+    public addChild(v: PomTree) {
+        this._children?.push(v);
+    }
+
+    public deepSearch(pomGav: string): PomTree | undefined {
+        if (this.pomGav === pomGav) {
+            return this;
+        }
+        for (const pom of this._children) {
+            pom.deepSearch(pomGav);
+        }
+        return;
+    }
+    public runMavenDependencyTree(): void {
+        MavenUtils.executeMavenCmd(`mvn dependency:tree -DappendOutput=true -DoutputFile=.jfrog_vscode/maven`, this.pomPath);
```

Comment:
```suggestion
        MavenUtils.executeMavenCmd(`mvn dependency:tree -DappendOutput=true -DoutputFile=.jfrog/maven`, this.pomPath);
```

Reply:
I don't want to name the folder '.jfrog' because we already use it by the cli.